style.css

```diff
@@ -1283,4 +3048,71 @@ body {
     width: 20px;
     height: 20px;
   }
+
+  .second-page {
+    min-height: 573px;
+  }
+
+  .second-page__img {
+    margin-bottom: 3%;
+  }
+
+  .second__wrap {
+    margin-bottom: 12px;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #EBE3D8 98px, #000 98px, #000 137px, #EBE3D8 137px);
+  }
+
+  .third-page__wrap {
+    width: 53%;
+    bottom: 26px;
+    margin-bottom: 39px;
+  }
+
+  .third-page__article {
+    padding: 22px 8px 26px 22px;
+    margin-bottom: 0;
+    width: 137%;
+  }
+
+  .third-page__title {
+    font-size: 20px;
+    margin-bottom: 2px;
+  }
+
+  .third-page__subtitle {
+    margin-bottom: 8px; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)